### ISSUE TITLE
solidity-coverage v0.4.1 conflicts with truffle v4.0.1

### ISSUE THREAD

**Issue by SevenSwen** [sc-forks/solidity-coverage#155]
# Issue
solidity-parser-sc generated an exception in a simple project, which uses truffle v4.0.1

# Steps to Reproduce
- truffle init
- npm init
- npm i --save zeppelin-solidity
- npm i --save-dev solidity-coverage
- touch ./contracts/MyCoin.sol
- touch .solcover.js
- npm run coverage

./contracts/MyCoin.sol:
```
pragma solidity ^0.4.18;

import 'zeppelin-solidity/contracts/token/StandardToken.sol';

contract MyCoin is StandardToken {
    string constant public name = "MyCoin";
    string constant public symbol = "MYC";
    uint8 constant public decimals = 18;
}
```
.solcover.js:
```
module.exports = {
  copyNodeModules: true
};
```
see details https://github.com/SevenSwen/test-solidity-coverage
# Expected Behavior
usual solidity-coverage report on test coverage

# Actual Results
![Error](https://raw.githubusercontent.com/SevenSwen/test-solidity-coverage/master/Screen%20Shot%202017-11-20%20at%2015.11.50.png)

# Environment
truffle v4.0.1
solidity-coverage v0.4.1
zeppelin-solidity v1.3.0
npm 5.5.1

**Comment by area** [comment]
Thanks for this report, and especially the reproduction steps. I've identified the problem, and should be able to fix it shortly.

**Comment by cgewecke** [comment]
Published `0.42` - Thanks @area for the fix. Thanks @SevenSwen for reporting.
